suite-native/discovery/src/discoveryConfigSlice.ts

```diff
@@ -0,0 +1,29 @@
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+
+export interface DiscoveryConfigState {
+    areTestnetsEnabled: boolean;
+}
+
+export type DiscoveryConfigSliceRootState = {
+    wallet: { discoveryConfig: DiscoveryConfigState };
```

Comment:
Not sure if this needs to be added in the wallet namespace. I'd probably just stick it into the root of our state, but let's discuss reasons. Discovery is in wallet not to break suite state, but since this is more about app setup, I'd put it in root instead of wallet

Reply:
moved to the root state in 4a8d956e6d